clang/lib/AST/ByteCode/InterpBuiltin.cpp

```diff
@@ -2817,6 +2817,81 @@ static bool interp__builtin_select(InterpState &S, CodePtr OpPC,
   return true;
 }
 
+static bool interp__builtin_elementwise_fsh(InterpState &S, CodePtr OpPC,
+                                            const CallExpr *Call,
+                                            unsigned BuiltinID) {
+  assert(Call->getNumArgs() == 3);
+
+  const QualType Arg0Type = Call->getArg(0)->getType();
```

Comment:
```suggestion
  QualType Arg0Type = Call->getArg(0)->getType();
```
Sorry, forgot that the last time.